content/en/docs/duplicate-certificate-limit.md

```diff
@@ -23,6 +24,31 @@ set" of hostnames for your certificate would be `[example.com]`. This rate limit
 is exceeded when a subscriber requests a certificate for the same "exact set" of
 hostnames more than 5 times in a single week.
 
+## Wait for Reset
+
+Revoking the previously issued certificates will not reset the Duplicate
+Certificate limit. Subscribers can request another certificate 168 hours (7
+days) after the  most recent successful issuance. If you're unsure of the date
```

Comment:
I believe this should be from their first of 5 issuances in a 168 hour window. If they issue one cert on Tuesday and 4 while fiddling on Wednesday, the next opportunity to issue a certificate is 168 hours from the Tuesday issuance.

Reply:
You're absolutely correct, I mis-remembered this implementation. I'll reword this.